android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/SplashActivity.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.ui.login
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.lifecycle.lifecycleScope
+import com.google.android.fhir.sync.State
+import com.google.android.fhir.sync.Sync
+import kotlinx.coroutines.InternalCoroutinesApi
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import org.smartregister.fhircore.anc.AncApplication
+import org.smartregister.fhircore.anc.R
+import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
+import org.smartregister.fhircore.engine.ui.base.BaseMultiLanguageActivity
+import org.smartregister.fhircore.engine.util.extension.showToast
+import timber.log.Timber
+
+class SplashActivity : BaseMultiLanguageActivity() {
```

Comment:
Not required after login. We may need a branded **Welcome Screen** when the application is launching. @f-odhiambo can comment

Reply:
Like I suggested we can have the branded Welcome Screen shown immediately when the app is launched instead of showing it between login and register screens. We will need to implement this UI as `ConfigurableView` in engine module `ui/welcomescreen` package. @f-odhiambo is this something that we can request the design team for?

Reply:
With regards to  Welcome Screen when the application is launching, his ideally was not agreed upon and might need approval from @roger. I think we can remove it for now. I will raise it up and n implementation can be done in a separate ticket with agreed-upon designs/mocks